docs/rand.rst

```diff
@@ -17,3 +17,4 @@ Random
     toqito.rand.random_state_vector
     toqito.rand.random_states
     toqito.rand.random_unitary
+    toqito.rand.random_circulant_gram
```

Comment:
minor nit: might want to move this addition up (alphabetical order)

Reply:
done